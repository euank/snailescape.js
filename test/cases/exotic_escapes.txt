\a\b\e\E\f\r\t\v\\ "\a\b\e\E\f\r\t\v\\"
