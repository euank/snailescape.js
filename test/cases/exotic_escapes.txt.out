["\u0007\b\u001b\u001b\f\r\t\u000b\\","\u0007\b\u001b\u001b\f\r\t\u000b\\"]
